app/views/species_list/bulk_editor.html.erb

```diff
@@ -94,19 +94,21 @@
     <%=
       args = { controller: :observations, action: :destroy, id: "XXX" }
       url = url_for(add_query_param(args, @query))
+      # NOTE: This doesn't work with new obs CRUD. Needs to be destroy button
```

Comment:
Errr...  Am I missing something, or is this action dead?  I don't see it at all in the controller.  There's still a route to bulk_editor, but there's no corresponding method in the controller, and without that how is +@species_list+ being initialized for the view...  and doesn't it have to POST at some point, anyway?  I feel like this was ripped out some time ago, but apparently not completely.

Reply:
BTW, I see 6 attempts to load it in 2020, the latest two on 20200813/4, then nothing.  I say we remove this.

Reply:
Deleted